tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -350,17 +315,13 @@ def update_job_with_s3_inventory_in_postgres(
             )
             # Update job status
             LOGGER.debug(f"Posting successful status for job {job_id}.")
-            connection.execute(
-                update_job_sql(),
-                [
-                    {
-                        "id": job_id,
-                        "status_id": OrcaStatus.STAGED.value,
-                        "last_update": datetime.now(timezone.utc).isoformat(),
-                        "end_time": None,
-                        "error_message": None,
-                    }
-                ],
+            orca_shared.reconciliation.shared_reconciliation.update_job(
```

Comment:
See comments on possible signature changes to this function call.

Reply:
Fixed